doc/Overview/Reactive/How-tos/SimpleFeed.md

```diff
@@ -0,0 +1,104 @@
+        }
+    }
+    ```
+
+    We're using [records](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/record) in purpose,
+    as records are immutable and ensure purity of objects. Records also implements easy equality comparison and hashing.
+
+1. Create a file named *WeatherModel.cs* replacing its content with the following:
+
+    ```c#
+    public partial record WeatherModel
+    {
+        private readonly WeatherService _weatherService = new();
+    
+        public IFeed<WeatherInfo> CurrentWeather => Feed.Async(_weatherService.GetCurrentWeatherAsync);
+    }
+    ```
+
+    MVUX's analyzers will read the `WeatherModel` and will generate a special mirrored `BindableWeatherModel`,
+    which provides binding capabilities for the View, so that we can stick to sending update message in an MVU fashion.
+    
+    The `CurrentWeather` property value also gets cached, so no need to worry about its being created upon each `get`.
+    
+    <!-- TODO the generated code can be inspected via project->analyzers etc. -->
+
+
+1. Open the file `MainView.xaml` and add the following namespace to the XAML:
+
+    `xmlns:mvux="using:Uno.Extensions.Reactive.UI"`
+
+1. Replace anything inside the `Page` element with the following code:
+
+    ```xaml
+    <mvux:FeedView Source="{Binding CurrentWeather}">
+        <DataTemplate>
+            <StackPanel>
+                <TextBlock DataContext="{Binding Data}" Text="{Binding Temperature}"/>
+                <Button Content="Refresh" Command="{Binding Refresh}" />
+            </StackPanel>
+        </DataTemplate>
+
+        <!-- Optional: this will show up when loading, or when we click refresh and data is still loading -->    
+        <mvux:FeedView.ProgressTemplate>
+            <DataTemplate>            
+                <TextBlock Text="Requesting temperature..."/>
+            </DataTemplate>
+        </mvux:FeedView.ProgressTemplate>
+    </mvux:FeedView>
+    ```
+
+    The `FeedView` wraps its source (in this case our `CurrentWeather` feed) in a `FeedViewState` object
+    that provides the `Data` property to access the data currently available in the `CurrentWeather` feed,
+    the `Refresh` command that explicitly triggers reloading the trip.
+
+1. Press <kbd>F7</kbd> to navigate to open code-view, and in the constructor, after the line that calls `InitializeComponent()`, add the following line:
+
+    ```c#
+    this.DataContext = new BindableWeatherModel();
+    ```
+
+    The `BindableWeatherModel` is a special MVUX-generated mirror class that represents a mirror of the `WeatherModel` adding binding capabilities,
+    for MVUX to be able to recreate and renew the model when an update message is sent by the view.  
+    
+1. Click <kbd>F5</kbd> to run the project
+
+1. When the app loads you'll notice how the `ProgressTemplate` shows (if you've included one), till the data is received from the service (2 seconds).
```

Comment:
At this point the only thing the user should have seen is
- Model
- Databinding via FeedView
- Creating instance of the BindableWeatherModel

Comment that this is similar to how you might data bind with MVVM. However, we have much more thanks to IFeed..... for example progress state for when data is loading (and talk through using ProgressTemplate) and ability to refresh the data (and talk through adding Refresh command).